Tribler/Test/Community/Market/test_message.py

```diff
@@ -21,7 +21,7 @@ def test_init(self):
 
     def test_conversion(self):
         # Test for conversions
-        self.assertEqual('0', str(self.trader_id))
+        self.assertEqual(str(b'0'), str(self.trader_id))
```

Comment:
Instead of casting to `str`, you should test for the `to_string` method I recently added to `TraderId`.

Reply:
The tests pass on Python 2 but what will happen on Python 3?